### DISCUSSION TITLE
Scraping Parameters Not Set Up!

### DISCUSSION THREAD

**Discussion by tampa3** [CodyBerenson/PGMA-Modernized#248] [General]
I saw a post earlier about this issue but can no longer find it,  and I cannot seem to get anything to match. I just downloaded the latest plugins and on current Version 1.32.2.7100.  

I was also getting this error with 1.31.3.6868 as well. I was wondering what the solution was?

[com.plexapp.agents.GEVI.log](https://drive.google.com/file/d/1RvQCY3MXu9yBBXIvoPNblcEuZCB27l9z/view?usp=sharing)

[com.plexapp.agents.GayAdultFilms.log](https://drive.google.com/file/d/1wm8Z0HsQypj3W1vpqdfEQSQqI7As9vJk/view?usp=sharing)

[com.plexapp.agents.GayEmpire.log](https://drive.google.com/file/d/1QA5b7HXDf68GNLj_1d1rv2vY_cna7hWF/view?usp=sharing)

**Comment by CodyBerenson** [comment]
Per the Installation instructions:  

When looking at your log, one can see that you have not made a selection for genrecollection.  You actually have to choose one.  It may appear that Indigo is the default.  It is not as you can see here:

![image](https://github.com/CodyBerenson/PGMA-Modernized/assets/40373801/998ebffc-ddb5-4a34-adc9-def52279587c)

The solution is simple:  follow the instructions under installation on the main page  

https://github.com/CodyBerenson/PGMA-Modernized#installation-and-usage

**Comment by JPH71** [comment]
As Cody has pointed out - You have not set your preferences... This as you
could see was highlighted in the log files. A version or 2 back I enhanced
this section of the logs so that it was easily discernible.

Jason xxx

**Comment by tampa3** [comment]
OMG I am such a goof.  I didn't include the _PGMA folder in plugins because I didn't consider that a bundle.

**Comment by JPH71** [comment]
You need to do a plex dance... set it to no... quit pms... set it to
another colour quit... then back to NO...

**Comment by Charlotte-br560** [comment]
It seems like you're encountering an error with certain Plex plugins. The issue might not be directly related to the versions you mentioned, but rather the plugins themselves. Have you tried checking the plugin configurations or reinstalling them?

**Reply by tampa3** [reply]
The previous suggestions worked for me, Setting preferences. Everything has been working perfect ever since.

**Comment by JPH71** [comment]
Thanks for letting us know...
Hugs x
